docs/gaia-tutorials/join-mainnet.md

```diff
@@ -70,9 +69,9 @@ You can edit the `~/.gaia/config/app.toml` file in order to enable the anti spam
 
 # The minimum gas prices a validator is willing to accept for processing a
 # transaction. A transaction's fees must meet the minimum of any denomination
-# specified in this config (e.g. 10uatom).
+# specified in this config (for example, 10uatom).
 
-minimum-gas-prices = ""
+minimum-gas-prices = "0.025uatom"
```

Comment:
Yes, it's actually in the same document a few lines below 

End of chapter `a note on gas and fees`. So I thought let's make it even more clear. Having no input is actually dangerous (because of transaction flood) - so let's add it to the code snipped directly.
https://github.com/cosmos/gaia/blob/main/docs/gaia-tutorials/join-mainnet.md#a-note-on-gas-and-fees

Reply:
Speaking about this, we could consider making this default into the bootstrapped `app.toml` file.